Drivers/RcReceiver/Inc/CommonDataTypes.h

```diff
@@ -0,0 +1,93 @@
+#ifndef COMMON_DATA_TYPES_H
```

Comment:
Maybe this file can be renamed to something specific to rc? If other classes outside these drivers need access to this maybe it should be moved out of the RCReceiver folder?